src/lib.rs

```diff
@@ -66,6 +87,53 @@ impl VehiclePropertyValue {
         }
     }
 
+    fn from_prop(
+        vhal_prop_type: &VehiclePropertyType,
+        prop_value: &VehicleHalProto::VehiclePropValue,
```

Comment:
Maybe nitpicking:
Worth both parameters either having or not having the `vhal` prefix?

Reply:
Yes, consistency is better. But I'll change it the other way around. I think `vhal_` prefix can be omitted, the whole library is about vhal stuff!